tests/test_parse.c

```diff
@@ -446,7 +446,7 @@ static void test_incremental_parse()
 		json_tokener_set_flags(tok, step->tok_flags);
 
 		if (length == -1)
-			length = strlen(step->string_to_parse);
+			length = (int)strlen(step->string_to_parse);
```

Comment:
How about changing `int length` to be size_t instead?

Reply:
That doesn't really make sense, given that size_t is unsigned and the value -1 is expected.

Reply:
Oh, right.  never mind then, let's leave it with the cast to int in place then.